lib/matplotlib/axes/__init__.pyi

```diff
@@ -1,10 +1,12 @@
 from typing import TypeVar
 
-from ._axes import *
-from ._axes import Axes as Subplot
+from ._axes import Axes as Axes
```

Comment:
Since that's the only thing in there, should we change the import in the `.py` file, too?

Reply:
I mean, logically yes, but technically at runtime more gets imported there, just not things we _intend_ to re-export.

I was being a bit conservative here and _only_ changing stub behavior (especially since the stubs _didn't_ pick up `Subplot` as being exported, when it _should_ be)

Reply:
I actually think `__init__.py` files (broadly) are good candidates for inlining type hints and just not having separate files: they are short, contain little to no actual function definitions, it's mostly imports and `__all__` (which I'm also in favor of adding in more places)